Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -9128,6 +9128,102 @@ def _extract_integration_display_name(_integration_brand):
     return pack_version
 
 
+def create_indicator_result_with_dbotscore_unknown(indicator, indicator_type, reliability=None,
+                                                   context_prefix=None, address_type=None):
+    '''
+    Used for cases where the api response to an indicator is not found,
+    returns CommandResults with readable_output generic in this case, and indicator with DBotScore unknown
+
+    :type indicator: ``str``
+    :param name: The value of the indicator
+
+    :type indicator_type: ``DBotScoreType``
+    :param indicator_type: use DBotScoreType class [Unsupport in types CVE and ATTACKPATTERN]
+
+    :type reliability: ``DBotScoreReliability``
+    :param reliability: use DBotScoreReliability class
+
+    :type context_prefix: ``str``
+    :param context_prefix: Use only in case that the indicator is CustomIndicator
+
+    :type address_type: ``str``
+    :param address_type: Use only in case that the indicator is Cryptocurrency
+
+    :rtype: ``CommandResults``
+    :return: CommandResults
+    '''
+    if not context_prefix and (indicator_type is DBotScoreType.CUSTOM or not DBotScoreType.is_valid_type(indicator_type)):
+        raise ValueError('Indicator type is invalid')
+
+    #  There is no justification to support the following two types, because they have a fixed dbotscore.
+    if indicator_type in [DBotScoreType.CVE, DBotScoreType.ATTACKPATTERN]:
```

Comment:
It's best to put these types of comments under the condition, but I also took to liberty to:
1. Change list->tuple. In general it's a best practice to use `tuple` when you're creating a static immutable collection.
2. Streamlined the comment.
```suggestion
    if indicator_type in (DBotScoreType.CVE, DBotScoreType.ATTACKPATTERN):
    #  not supportted, because they have a fixed dbotscore
```